common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -36,8 +36,11 @@
                 }
                 return Problem.prototype.enableSubmitButton.apply(that, arguments);
             };
-            this.enableAllButtons = function(enable, isFromCheckOperation) { // eslint-disable-line no-unused-vars
-                return Problem.prototype.enableAllButtons.apply(that, arguments);
+            this.enableButtons = function() {
```

Comment:
This style is a remnant of the conversion from CoffeeScript to JavaScript. Could you change this so that the methods declare their arguments, and then just pass the arguments explicitly to the prototype functions?